vercel.json

```diff
@@ -1493,7 +1493,7 @@
     },
     {
       "source": "/docs/guides/deployment/deployment-guides/use-prisma-with-serverless-framework",
-      "destination": "/docs/guides/deployment/use-prisma-with-serverless-framework"
```

Comment:
We also need a redirect from this URL to the new one if this was live in the past, and it seems it was.